src/js/components/Tabs/Tabs.js

```diff
@@ -93,37 +180,145 @@ const Tabs = forwardRef(
       messages,
     })}`;
 
+    const onLeft = () => {
+      if (focusIndex !== 0) {
+        tabRefs[focusIndex - 1].current.focus();
+        setFocusIndex(focusIndex - 1);
+      }
+    };
+
+    const onRight = () => {
+      if (focusIndex < tabRefs.length - 1) {
+        tabRefs[focusIndex + 1].current.focus();
+        setFocusIndex(focusIndex + 1);
+      }
+    };
+
+    const onHome = () => {
+      if (focusIndex !== 0 && tabRefs.length > 0) tabRefs[0].current.focus();
+    };
+
+    const onEnd = () => {
+      if (focusIndex !== 0 && tabRefs.length > 0)
+        tabRefs[tabRefs.length - 1].current.focus();
+    };
+
     return (
-      <StyledTabs
-        ref={ref}
-        as={Box}
-        role="tablist"
-        flex={flex}
-        responsive={responsive}
-        {...rest}
-        background={theme.tabs.background}
-      >
-        <StyledTabsHeader
+      <Keyboard onLeft={onLeft} onRight={onRight} onHome={onHome} onEnd={onEnd}>
+        <StyledTabs
+          ref={ref}
           as={Box}
-          direction="row"
-          justify={justify}
-          alignSelf={alignControls}
-          flex={false}
-          wrap
-          background={theme.tabs.header.background}
-          gap={theme.tabs.gap}
-          {...tabsHeaderStyles}
-        >
-          {tabs}
-        </StyledTabsHeader>
-        <StyledTabPanel
+          role="tablist"
           flex={flex}
-          aria-label={tabContentTitle}
-          role="tabpanel"
+          responsive={responsive}
+          {...rest}
+          background={theme.tabs.background}
         >
-          {activeContent}
-        </StyledTabPanel>
-      </StyledTabs>
+          <Box direction={overflow && scroll ? 'row' : 'column'}>
+            {overflow && scroll && (
+              <Button
+                a11yTitle="Previous Tab"
```

Comment:
I'm wondering if these buttons should even be in the tab flow for a keyboard user because for example:
1. if the active tab is Tab 1
2. then I tab to the "Next" button and advance so I'm seeing Tab 10 (for example)
3. I want to select that, I have to shift+tab to back to the tabs
4. Tab 1 is active so now I have to use arrow keys to move to Tab 10

Propose that we either mimic the behavior shown in this [Angular responsive tabs example](https://material.angularjs.org/latest/demo/tabs) OR make it so "shift+tab" from the "Next" button places focus on the farthest right visible tab and "tab" from the "Previous button takes you to the farthest left visible tab?

Demo of current behavior:
https://user-images.githubusercontent.com/12522275/169605670-3fe98fcd-40da-473b-867e-794ebf8cff1a.mov

Reply:
I'm supportive of removing the left and right arrows from the tabIndex. It feels like they add an extra element that doesn't provide much utility for keyboard only users.

Reply:
Set `tabIndex={-1}` for the left and right arrow in the latest commit